pkg/kubelet/cm/deviceplugin/device_plugin_handler.go

```diff
@@ -337,19 +343,17 @@ func (h *HandlerImpl) writeCheckpoint() error {
 	if err != nil {
 		return err
 	}
-	filepath := h.devicePluginManager.CheckpointFile()
-	return ioutil.WriteFile(filepath, dataJSON, 0644)
+	return h.store.Write(defaultCheckpoint, dataJSON)
```

Comment:
better check the return value although the chance of failure is small.

Reply:
Looks to me thats already being taken care of up in the call tree, in the Allocate function. L 346 would return nil or err.